cognite/client/data_classes/data_modeling/ids.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional, Sequence, Tuple, Union, cast
+
+from cognite.client.utils._identifier import DataModelingIdentifier, DataModelingIdentifierSequence
+
+
+@dataclass
+class DataModelingId:
+    space: str
+    external_id: str
+
+    @classmethod
+    def from_tuple(cls, tup: tuple[str, str] | tuple[str, str, str]) -> DataModelingId:
+        return DataModelingId(*tup)
+
+
+@dataclass
+class VersionedDataModelingId:
+    space: str
+    external_id: str
+    version: Optional[str] = None
+
+    @classmethod
+    def from_tuple(cls, tup: tuple[str, str] | tuple[str, str, str]) -> VersionedDataModelingId:
+        return VersionedDataModelingId(*tup)
+
+
+ContainerId = Union[DataModelingId, Tuple[str, str]]
+ViewId = Union[VersionedDataModelingId, Tuple[str, str], Tuple[str, str, str]]
+DataModelId = Union[VersionedDataModelingId, Tuple[str, str], Tuple[str, str, str]]
```

Comment:
```suggestion
ViewId = Union[VersionedDataModelingId, Tuple[str, str], Tuple[str, str, str]]
DataModelId = ViewId
```

Reply:
This gives the impression that DataModelId is the same as ViewId, which it is not, even though it has the same type. So suggest keeping it.